sound/soc/sof/intel/pci-tgl.c

```diff
@@ -247,6 +253,7 @@ static const struct sof_dev_desc rpl_desc = {
 	.chip_info = &tgl_chip_info,
 	.ipc_supported_mask	= BIT(SOF_IPC) | BIT(SOF_INTEL_IPC4),
 	.ipc_default		= SOF_IPC,
+	.dspless_mode_supported	= true,		/* Only supported for HDaudio */
```

Comment:
any reason why we only add this mode for APL and TGL? CNL/ICL/MTL are missing.

Reply:
I only tested on APL and TGL, but I will allow DSPless mode for the other platforms having HDA.